### ISSUE TITLE
Torrents are not added to qBittorrent but listed as Accepted by Flexget

### ISSUE THREAD

**Issue by hhiru** [Flexget/Flexget#1863]
### Expected behaviour:
When a torrent is accepted by Flexget, qBittorrent starts downloading it.

### Actual behaviour:
The entry is accepted by Flexget but no torrent is added by qBittorrent.

### Steps to reproduce:
- Step 1: Wait for the schedule task to start and for flexget to execute (or manually execute it)
- Step 2: Check if the torrent has been accepted by Flexget
- Step 3: Check if the torrent has been listed in qBittorrent.

#### Config:
```
templates:
  global:
    qbittorrent:
      host: localhost
      port: 8080
    notify:
      entries:
        title: New episode downloading!
        what: accepted
        via:
          - toast: yes
  anime:
    regexp:
      reject:
        - batch
    qbittorrent:
      path: D:\MyPath\anime
  tvshow:
    qbittorrent: 
      path: D:\MyPath\shows
```
  
#### Log:
```
[...]
2017-06-02 20:10 VERBOSE  task          hs              ACCEPTED: `[HorribleSubs] Shingeki no Bahamut - Virgin Soul - 09 [720p].mkv` by series plugin because choosing best available quality
2017-06-02 20:10 INFO     download      hs              Downloading: [HorribleSubs] Shingeki no Bahamut - Virgin Soul - 09 [720p].mkv
2017-06-02 20:10 VERBOSE  details       hs              Summary - Accepted: 1 (Rejected: 15 Undecided: 59 Failed: 0)
2017-06-02 20:10 VERBOSE  notify        hs              Successfully sent a notification to `toast`
[...]
2017-06-03 17:10 VERBOSE  task          hs              ACCEPTED: `[HorribleSubs] Re-Creators - 09 [720p].mkv` by series plugin because choosing best available quality
2017-06-03 17:10 INFO     download      hs              Downloading: [HorribleSubs] Re-Creators - 09 [720p].mkv
2017-06-03 17:10 VERBOSE  details       hs              Summary - Accepted: 1 (Rejected: 10 Undecided: 64 Failed: 0)
2017-06-03 17:10 VERBOSE  notify        hs              Successfully sent a notification to `toast`
```

### Additional information:

- FlexGet version: 2.10.56
- Python version: 3.6.1
- Installation method: pip
- Using daemon (yes/no): no
- OS and version: Windows 10 1703
- Link to crash log:  [flexget.zip](https://github.com/Flexget/Flexget/files/1049726/flexget.zip)

**Comment by liiight** [comment]
Please paste debug logs and qbitorrent version

**Comment by hhiru** [comment]
Where are debug logs stored? Are they stored as normal logs in the .flexget folder?
Also I'm using the latest x64 version of QB (v 3.3.13)

**Comment by hhiru** [comment]
Here is the log [flexget.zip](https://github.com/Flexget/Flexget/files/1049743/flexget.zip)

**Comment by hhiru** [comment]
Look at line 1220
[flexget.zip](https://github.com/Flexget/Flexget/files/1049747/flexget.zip)

**Comment by hhiru** [comment]
Here's the debug log
[flexget.zip](https://github.com/Flexget/Flexget/files/1049749/flexget.zip)

**Comment by liiight** [comment]
Please use pastebin

**Comment by hhiru** [comment]
https://pastebin.com/L9piaKtB

**Comment by liiight** [comment]
Is it possible you have more than 1 instance of qbitorrent? Logs look fine.

**Comment by hhiru** [comment]
Only one process is active. What changed from when everything worked and now is that I'm using the x64 version of qB. I can try uninstalling and installing the x86 version and see if it works

**Comment by liiight** [comment]
Logs specifically say that a connection is made and file is uploaded successfully, so I don't think this is a flexget issue...

**Comment by hhiru** [comment]
Nope. I installed the x86 version and same problem. I have the two windows side by side, flexget accepts the entries but nothing happens to qbittorrent. I'll try installing a previous version now just in case

**Comment by hhiru** [comment]
Aaaaaand it worked! :O Now, I don't know if it worked because it is the previous version or because this time it installed the x64 version in Program Files instead of Program Files x86 like with the previous installer. I'll uninstall everything again and retry with the latest version and see if this time it installs everything in the proper path. I suspect there was some sort of issue with the path since I at first installed the x86 version that went in the Program Files x86 folder, then without uninstalling the x86 version I installed the x64.

Anyway I'll report back what happens!

**Comment by crawln45** [comment]
create a ticket here. https://github.com/qbittorrent/qBittorrent/issues for qbittorrent to look into.

**Comment by crawln45** [comment]
Told this should be fixed in 3.3.14
